src/wire/libevent_server.cpp

```diff
@@ -28,7 +28,7 @@ std::vector<std::unique_ptr<LibeventSocket>>
     &LibeventServer::GetGlobalSocketList() {
   static std::vector<std::unique_ptr<LibeventSocket>>
       // 2 fd's per thread for pipe and 1 listening socket
-      global_socket_list(FLAGS_max_connections + QUERY_THREAD_COUNT * 2 + 1);
+      global_socket_list(FLAGS_max_connections + CONNECTION_THREAD_COUNT * 2 + 1);
```

Comment:
magic numbers 
(I know you didn't add these but if you are already here, then this is a good time to fix them)